src/components/ContractInfo.tsx

```diff
@@ -32,6 +35,13 @@ const ContractInfo = () => {
         Fee
         <div className="pr-2">{displayPercent(fee, 2)}%</div>
       </InfoItem>
+      <InfoItem>
+        Reserve Balance
+        <div className="pr-2">
+          {displayAmount(anchorPoolBalance, 2)}{' '}
```

Comment:
should we displayAmount when the balance is undefined? consider rendering something to indicate the balance hasn't been received yet